dist/TempManager.js

```diff
@@ -0,0 +1,46 @@
+class TempManager {
+    constructor() {
+        this.cityData = []
+    }
+
+    async getDataFromDB() {
+        let cities = await $.get('/cities')
+        this.cityData = cities
+    }
+
+    async getCityData(cityName) {
+            const data = await $.get(`/city/${cityName}`)
+            if (!data.name) {
```

Comment:
try to be positive. In your `if()` statements try to no use `if(!somthing)` instead 
`if(data.name){`
`this.cityData.push(data)`
`else {`
               ` alert("You've entered an invalid city name")`
`}`